utilities/pipelines/dependencies/Microsoft.Network/virtualNetworks/parameters/8.aadds.parameters.json

```
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "adp-<<namePrefix>>-az-vnet-aadds-001"
+    },
+    "addressPrefixes": {
+      "value": [
+        "10.0.0.0/16"
```

The prefix number of the param file name is mapped to the address space. I'd suggest 
```suggestion
        "10.8.0.0/16"
```